test/e2e/build_test.go

```diff
@@ -0,0 +1,103 @@
+// +build !remoteclient
+
+package integration
+
+import (
+	"os"
+	"strings"
+
+	. "github.com/containers/libpod/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman build", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest *PodmanTestIntegration
+	)
+
+	BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanTestCreate(tempdir)
+		podmanTest.Setup()
+		podmanTest.RestoreAllArtifacts()
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+		f := CurrentGinkgoTestDescription()
+		processTestResult(f)
+
+	})
+
+	It("podman build basic alpine", func() {
```

Comment:
Mind changing the name and/or adding a quick comment for this test:   It took me a moment to realize the `rmi` (below) is a secondary test...verifying a ancestor image is removable.  It's a small thing, but might save someone else 2 minutes of head-scratching.

Reply:
That's interesting.  I basically created this test as a "simple as it can get" to make sure it would work build and did the rmi soley to clean stuff up after.  Will comment and/or rename.

Reply:
I tweaked the name and added a comment.